integration-tests/mtls-certificates/src/test/java/io/quarkus/it/vertx/CertificateRoleMappingTest.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.it.vertx;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.net.URL;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+
+@QuarkusTest
```

Comment:
Considering you are adding whole new module to integration tests (and not just junit test into extensions module) I think you should also add `@QuarkusIntegrationTest`

Reply:
@michalvavrik Good catch, I have it, forgot to add

Reply:
@michalvavrik I wanted to modify `integration-tests/vertx-http` but as I said it tests that only a single certificate is trusted and I needed at least two to test, which is why I thought of introducing a new integration tests module. I suppose having a native test for a feature request with Quarkus JIRA is worth it

Reply:
Sure, makes sense.